CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # v1.17
 
+## Release v1.17.1 - 2024/08/19
+
+### Enhancement
+
+* yaml_parser: Support $log_level element
+  https://github.com/fluent/fluentd/pull/4482
+* out_file: Add warn message for symlink_path setting
+  https://github.com/fluent/fluentd/pull/4502
+* out_http: Add `compress gzip` option
+  https://github.com/fluent/fluentd/pull/4528
+* in_exec: Add `encoding` option to handle non-ascii characters
+  https://github.com/fluent/fluentd/pull/4533
+* in_tail: Add throttling metrics
+  https://github.com/fluent/fluentd/pull/4578
+* compat: Improve method call performance
+  https://github.com/fluent/fluentd/pull/4588
+* Add logger gem dependency for Ruby 3.5
+  https://github.com/fluent/fluentd/pull/4589
+* in_sample: Add `reuse_record` parameter to reuse the sample data
+  https://github.com/fluent/fluentd/pull/4586
+
+### Bug Fix
+
+* logger: Fix LoadError with console gem v1.25
+  https://github.com/fluent/fluentd/pull/4492
+* parser_json: Fix wrong LoadError warning
+  https://github.com/fluent/fluentd/pull/4522
+* in_tail: Fix to skip lines with max line size limitation
+  https://github.com/fluent/fluentd/pull/4530
```

Comment:
It is difficult to explain #4530 briefly..., but it should be more concrete for users to understand this fix.
How about this?

```suggestion
* in_tail: Fix an issue where a large single line could consume a large amount of memory even though `max_line_size` is set
  https://github.com/fluent/fluentd/pull/4530
```